Tests/tests/test_OCIContainer.cpp

```diff
@@ -75,40 +80,40 @@ TEST_F(OCIContainerTest, RegisteredMethods){
     EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("pauseContainer")));
     EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("resumeContainer")));
     EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("executeCommand")));
- }
-
- TEST_F(OCIContainerInitializedTest, listContainersTest){
+}
 
-    std::list<std::pair<int32_t, std::string>> containerslist = {{descriptor1, containerId1},{descriptor2, containerId2}};
+TEST_F(OCIContainerInitializedTest, listContainersTest)
+{
+    std::list<std::pair<int32_t, std::string>> containerslist = {{91, "com.bskyb.epgui"},{94, "Netflix"}};
     EXPECT_CALL(dobbymock, listContainers())
         .WillOnce(::testing::Return(containerslist));
 
     EXPECT_EQ(Core::ERROR_NONE, handler.Invoke(connection, _T("listContainers"), _T("{}"), response));
     EXPECT_EQ(response, string("{\"containers\":[{\"Descriptor\":91,\"Id\":\"com.bskyb.epgui\"},{\"Descriptor\":94,\"Id\":\"Netflix\"}],\"success\":true}"));
 }
 
- TEST_F(OCIContainerInitializedTest, getContainerStateTest){
-
-    std::list<std::pair<int32_t, std::string>> containerslist = {{descriptor1, containerId1},{descriptor2, containerId2}};
+TEST_F(OCIContainerInitializedTest, getContainerStateTest)
+{
+    std::list<std::pair<int32_t, std::string>> containerslist = {{91, "com.bskyb.epgui"},{94, "Netflix"}};
     EXPECT_CALL(dobbymock, listContainers())
         .WillOnce(::testing::Return(containerslist));
 
-    EXPECT_CALL(dobbymock, getContainerState(descriptor1))
-        .WillOnce(::testing::Return(stateRunning));
+    EXPECT_CALL(dobbymock, getContainerState(91))
+        .WillOnce(::testing::Return(ContainerState::Running));
 
     EXPECT_EQ(Core::ERROR_NONE, handler.Invoke(connection, _T("getContainerState"), _T("{"
         "\"containerId\": \"com.bskyb.epgui\"}"), response));
     EXPECT_EQ(response, string("{\"containerId\":\"com.bskyb.epgui\",\"state\":\"Running\",\"success\":true}"));
 }
 
 
- TEST_F(OCIContainerInitializedTest, getContainerInfoTest){
-
-    std::list<std::pair<int32_t, std::string>> containerslist = {{descriptor1, containerId1},{descriptor2, containerId2}};
+TEST_F(OCIContainerInitializedTest, getContainerInfoTest)
+{
+    std::list<std::pair<int32_t, std::string>> containerslist = {{91, "com.bskyb.epgui"},{94, "Netflix"}};
     EXPECT_CALL(dobbymock, listContainers())
         .WillOnce(::testing::Return(containerslist));
 
-    EXPECT_CALL(dobbymock, getContainerInfo(descriptor1))
+    EXPECT_CALL(dobbymock, getContainerInfo(91))
```

Comment:
Where do other magic numbers come from, for example "pids":[13132,13418] or "limit" : 419430400 ?

Reply:
These numbers I have being returned were taken from a real world example of what we expect to see being returned from this method. OCIContainer::getContainerInfo calls on DobbyProxy::getContainerInfo to return this information in a string